src/SFA.DAS.FAA.Application/SavedSearches/Commands/DeleteSavedSearches/DeleteSavedSearchesCommandHandler.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MediatR;
+using SFA.DAS.FAA.Data.SavedSearch;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.FAA.Application.SavedSearches.Commands.DeleteSavedSearches;
+
+public record DeleteSavedSearchesCommandHandler(ISavedSearchRepository SavedSearchRepository) : IRequestHandler<DeleteSavedSearchesCommand, Unit>
```

Comment:
Note: you don't need to use Unit as a return type, just don't implement a return type.